### PR TITLE
Update CakePHP framework guide

### PR DESCRIPTION
**Pull request by narendravaghela**
CakePHP Guide - #15

### CODE REVIEW

**Comment by danhunsaker** [comment]
Awesome. Let me know when you've made all the changes you've got in mind, and I'll review this one again/merge it in!

**Comment by narendravaghela** [comment]
Thanks @danhunsaker 
By the way. I am getting an error while setting up the Nanobox on my Windows machine. Below is the output. I selected Docker at the time of installation.
```
D:\Nanobox\nanobox-cakephp>nanobox run
Preparing environment :

--------------------------------------------------------------------------------
+ HEADS UP:
+ This is the first build for this project and will take longer than usual.
+ Future builds will pull from the cache and will be much faster.
--------------------------------------------------------------------------------


Building runtime :
  ! Pulling nanobox/build image

Error   : Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
Context : failed to build the code -> failed to pull the build image -> failed to pull docker image (nanobox/build)

Enter to continue:
```
Could you help with that please?

**Comment by danhunsaker** [comment]
I can give better help on Slack, but it appears the Hyper-V guest system actually running Docker for you (for those unaware, Docker relies on features in the Linux kernel to function, so there's always a VM involved if the host system isn't Linux already) is having connectivity issues. I imagine you've gotten this error fairly consistently when running it? Normally a retry or three will be enough to get past the issue.

**Comment by narendravaghela** [comment]
Hi @danhunsaker 
I am done with 2 steps: CakePHP from Scratch and Add a database.
I have tested them on Mac and Windows, and by following those steps, we can successfully setup our local CakePHP application.

Please review and suggest your inputs.

**Review by danhunsaker — APPROVED** [review]

File: articles/php/cakephp/from-scratch.md [hunk 1]
```diff
@@ -69,7 +79,7 @@ nanobox run php-server
 Visit your app at <a href="http://cakephp.dev" target="\_blank">cakephp.dev</a>
```
Comment (danhunsaker) [inline articles/php/cakephp/from-scratch.md:80]: `s/need develop/need to develop/`

**Review by HiteshSuthar7182 — COMMENTED** [review]
ys

**Pull request merged**